test/helpers/common.test.js

```diff
@@ -0,0 +1,66 @@
+import { CommonHelpers } from '../../helpers/common';
+import {fromFile, Parser} from '@asyncapi/parser';
+import path from 'path';
+
+const asyncapi_v3_path = path.resolve(__dirname, '../spec/asyncapi_v3.yml');
+const asyncapi_v2_path = path.resolve(__dirname, '../spec/asyncapi_v2.yml');
+
+const parser = new Parser();
+
+let v2Doc;
+let v3Doc;
+
+beforeAll(async () => {
+  const v2DocParsed = await fromFile(parser, asyncapi_v2_path).parse();
+  const v3DocParsed = await fromFile(parser, asyncapi_v3_path).parse();
+
+  v2Doc = v2DocParsed.document;
+  v3Doc = v3DocParsed.document;
+});
+
+describe('CommonHelpers', () => {
+  describe('v2', () => {
```

Comment:
for v2 imho we should still stick with old naming, so publish/subscribe to not confuse people

Reply:
good point. I adjusted code